src/controllers/trust.historical.beneficial.owner.controller.ts

```diff
@@ -0,0 +1,77 @@
+import { NextFunction, Request, Response } from 'express';
+import * as config from '../config';
+import { logger } from '../utils/logger';
+import { getApplicationData } from '../utils/application.data';
+import * as historicalBoMapper from '../utils/trust/historical.beneficial.owner.mapper';
+import { ApplicationData } from '../model/application.model';
+import * as PageModel from '../model/trust.page.model';
+import { TrustKey } from '../model/trust.model';
+
+const HISTORICAL_BO_TEXTS = {
+  title: 'Tell us about the former beneficial owner',
+};
+const get = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    const trustId = req.params['id'];
+    const pageData: PageModel.TrustHistoricalBeneficialOwnerPage = historicalBoMapper.mapTrustDetailToPage(
+      appData[TrustKey]?.find(trust => trust.trust_id === trustId),
+    );
+    console.log(pageData.trustName);
+
+    const templateName = config.TRUST_HISTORICAL_BENEFICIAL_OWNER_PAGE;
+    console.log("i am here");
+    console.log(`${config.TRUST_INVOLVED_URL}/${trustId}`);
+
+    return res.render(
```

Comment:
could you please make here like  in 

https://github.com/companieshouse/overseas-entities-web/pull/518/files
src/controllers/trust.details.controller.ts line 18

create page property type and getPageProps method

Reply:
We will take this on board during the next task. In this story no session mapping has been done, just getting page to render and URL paths to work on journey. We shouldn't need this until then